climate_scanner/trends_innovation_classifier/data_utils.py

```diff
@@ -53,6 +42,12 @@ def load_data():
 
 
 def pull_categoryData(inputData,category:str):
+    """
+    A function that filters a raw training dataset and generates a dataset for a specified category.
+    Arguments: inputData - jsonl file - List of dictionaries containing the raw training dataset 
+               category - Str - Free text of the specified category Ex: "Artificial Intelligence" 
+    :return: outputData - jsonl file - List of dictionaries containing the raw training dataset for the specified category
+    """
     outputData = []
     for i,d in enumerate(inputData):
         if category in d['category']:
```

Comment:
Just to ensure the function doesn't break - we have to ensure the name category is part of the Json (input) else it will through an error!